src/app/app.module.ts

```diff
@@ -3,14 +3,16 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { HomeComponent } from './pages/home-component/home/home.component';
+import { HomeComponent } from './pages/home/home.component';
 import { ScrollAnimationDirective } from './directives/scroll-animation.directive';
+import { PeriscopeComponent } from './components/periscope/periscope.component';
```

Comment:
Please sort the imports in asc mode

Reply:
Done. But shouldn't be this set in the tslint config file?

Reply:
Yeah, i will add this rule later